libs/signal-store/src/lib/modules/component-store.module.ts

```diff
@@ -0,0 +1,14 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { ComponentStoreConfig } from '@mini-rx/common';
+import { globalCsConfig } from '../component-store';
+
+@NgModule()
+export class ComponentStoreModule {
```

Comment:
Just a question. Can the ComponentStore be used in a standalone component (non module based app?)

Reply:
Sure, that is possible.

You can use ComponentStore wherever you want: classic components, standalone components.

ComponentStoreModule is only needed to configure all your ComponentStores globally. But that is optional.

In a classic module based app you would use `ComponentStoreModule.forRoot()`.
If you use modern Angular standalone APIs you would rather use `provideComponentStoreConfig`.